responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.component.ts

```diff
@@ -105,15 +119,58 @@ export class EvacueeSearchResultsComponent implements OnInit {
   }
 
   openWizard(): void {
-    this.cacheService.set(
-      'wizardOpenedFrom',
-      '/responder-access/search/evacuee'
-    );
-    this.evacueeSessionService.setWizardType(WizardType.NewRegistration);
+    if (this.evacueeSessionService.paperBased === true) {
```

Comment:
remove === true from condition, boolean resolves to true/false

Reply:
fixed!